### PR TITLE
Ajaxify version dropdown on source files page

### PR DESCRIPTION
**Pull request by orangejenny**
Branched off of https://github.com/dimagi/commcare-hq/pull/23242 , only the last commit is new. Once this is merged I'll use the same code to add a version dropdown to https://github.com/dimagi/commcare-hq/pull/23214

@mkangia 
code buddy @dannyroberts 

screen shot 2019-02-18 at 7 46 28 pm

### CODE REVIEW

File: corehq/apps/app_manager/static/app_manager/js/source_files.js [hunk 1]
```diff
@@ -10,36 +10,44 @@ hqDefine('app_manager/js/source_files',[
...
             if (!version) {
-                alert("Please enter a version to compare");
-                return;
-            } else if (!versionMap[version]) {
-                alert(version + " is not a valid version");
+                alert(gettext("Please enter a version to compare"));
                 return;
             }
-            window.location = initialPageData.reverse('diff', versionMap[version].build_id);
+            window.location = initialPageData.reverse('diff', version);
```
Comment (mkangia) [inline corehq/apps/app_manager/static/app_manager/js/source_files.js]: `version` now here can be also a part of a comment and not just number like `10 comment`?
would that be an issue here?

Comment (orangejenny) [reply corehq/apps/app_manager/static/app_manager/js/source_files.js]: No, `version` isn't the query itself, it's the value of the select, which is a build id. I should rename this variable.

Comment (orangejenny) [reply corehq/apps/app_manager/static/app_manager/js/source_files.js]: (updated)

File: corehq/apps/app_manager/views/download.py [hunk 2]
```diff
@@ -408,7 +408,6 @@ def download_index(request, domain, app_id):
             ),
             extra_tags='html'
         )
-    built_versions = get_all_built_app_ids_and_versions(domain, request.app.copy_of)
```
Comment (mkangia) [inline corehq/apps/app_manager/views/download.py]: 👍

**Comment by orangejenny** [comment]
Closing in favor of https://github.com/dimagi/commcare-hq/pull/23214

**Pull request closed without merging**